pkg/operation/certmanagement/certmanagment.go

```diff
@@ -26,13 +26,13 @@ var _ DNSProviderConfig = (*CloudDNSConfig)(nil)
 
 const (
 	// Route53 is a constant string for aws-route53.
-	Route53 DNSProvider = "aws-route53"
+	Route53 DNSProviderDeprecated = "aws-route53"
 	// CloudDNS is a constant string for google-clouddns.
-	CloudDNS DNSProvider = "google-clouddns"
+	CloudDNS DNSProviderDeprecated = "google-clouddns"
 )
 
-// DNSProvider returns the provider type  in-use.
-func (r *Route53Config) DNSProvider() DNSProvider {
+// DNSProviderDeprecated returns the provider type  in-use.
```

Comment:
Deprecated functions should have `// DEPRECATED` in their comment (see the various `kubernetes.Interface` methods in our codebase).